### ISSUE TITLE
EBML Cluster Finder Loop 

### ISSUE THREAD

**Issue by s0try** [wingman-jr-addon/wingman_jr#167]
Hello, thanks for working on such a neat project. I was familiarizing myself with the code and came upon this function to find the clusters in a WEBM file.
https://github.com/wingman-jr-addon/wingman_jr/blob/0deb0a7b4fa32c7086a8949fafc343305c961961/ebml.js#L212-L223
Since this function is called with indexes greater than 0, it would seem that for all calls where index is greater than 0, u8Array[i] is being compared to undefined since i values larger than 3 will be out of bounds for the EBML_CLUSTER_ID_ARRAY array. 

Fixing this function is trivial, but I am unsure how fixing it would affect the program as a whole, so I just wanted to bring it to your attention.

**Comment by wingman-jr-addon** [comment]
Yep, you're definitely right that's a bug. This branch of code only affects how the YouTube scanning of WebM works. Fortunately the impact is (I think) relatively low: the vast majority of videos I've browsed on YouTube seem to serve up as MP4, and I believe in at least some cases we additionally got lucky because it is not uncommon for the cluster to actually be at 0.
At any rate, thanks @s0try ! Fixed here: https://github.com/wingman-jr-addon/wingman_jr/commit/3b0ab301764df5e08505803b83203f270d1826a7
